src/app/examples/button-example/button-example.component.html

```diff
@@ -76,15 +76,58 @@ <h3>Medium (default)</h3>
   <label>Disabled</label>
 </div>
 
+<h3>Custom Icon</h3>
+<div class="button-example-container">
+  <button kirby-button>
+    <kirby-icon customName="football"></kirby-icon>
+  </button>
+  <label>Attention Level 1<br/><em>(Default)</em></label>
+</div>
+<div class="button-example-container">
+  <button kirby-button attentionLevel="2">
+    <kirby-icon customName="football"></kirby-icon>
+  </button>
+  <label>Attention Level 2</label>
+</div>
+<div class="button-example-container">
+  <button kirby-button attentionLevel="3">
+    <kirby-icon customName="football"></kirby-icon>
+  </button>
+  <label>Attention Level 3</label>
+</div>
+<div class="button-example-container">
+  <button kirby-button attentionLevel="4">
+    <kirby-icon customName="football"></kirby-icon>
+  </button>
+  <label>Attention Level 4</label>
+</div>
+<div class="button-example-container">
+  <button kirby-button disabled>
+    <kirby-icon customName="football"></kirby-icon>
+  </button>
+  <label>Disabled</label>
+</div>
+
 <h2>Button with icon and text</h2>
-<button kirby-button text="Icon Left">
+<button kirby-button>
   <kirby-icon name="edit"></kirby-icon>
+  Icon Left
 </button>
-<button kirby-button text="Icon Right" iconPlacement="right">
+<button kirby-button>
+  Icon Right
   <kirby-icon name="arrow-down"></kirby-icon>
 </button>
-<button kirby-button text="Custom Icon">
+<button kirby-button>
+  <label>Label Left</label>
```

Comment:
Holy crap, it's never dawned on me that labels can be used in buttons, I've always seen them as something you use for labeling `input`-, `select`- and `textarea`-elements.

I do however like it:
![giphub](https://media3.giphy.com/media/aAkNru67Hh40E/giphy.gif)

Reply:
I don't think it is w3c valid markup - if that is thing in anymore :D And it seems we need to add a cursor behaviour in this case otherwise the cursor will not highlight the text as clickable inside the button.

Reply:
> I don't think it is w3c valid markup - if that is thing in anymore :D And it seems we need to add a cursor behaviour in this case otherwise the cursor will not highlight the text as clickable inside the button.

Good point, I'll remove it, we should promote weird/non standard practices in the cookbook